middleware/i18n.js

```diff
@@ -15,7 +16,14 @@ const middleware = async (req, res, next) => {
 
 	if (currentLanguage) {
 		changeLanguage(currentLanguage);
-		res.cookie('USER_LANG', currentLanguage);
+		res.cookie('USER_LANG', currentLanguage, {
```

Comment:
create cookie helper setCookie(res, key, value, [options]) and reuse it for jwt cookie too

Reply:
https://github.com/hpi-schul-cloud/schulcloud-client/blob/97f8e05c9b513479bfdae46efe24905835db7d2c/controllers/registration.js#L151 and https://github.com/hpi-schul-cloud/schulcloud-client/blob/0004f582b7b905dd3dede2c5ae6309d9bbdf5e9c/helpers/authentication.js#L252-L253 should use it

Reply:
here, use the helper too :)